rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/RDFHandlerWrapper.java

```diff
@@ -42,8 +47,22 @@
 	 *                    <tt>null</tt>.
 	 */
 	public RDFHandlerWrapper(RDFHandler... rdfHandlers) {
+		this(new TreeModelFactory(), rdfHandlers);
```

Comment:
Does the RDFHandlerWrapper need a sorted Model? If not it should be swapped for the hash model.

Reply:
That would depend on the type of wrapper I think. This is just a default, specific implementations can override (ideally, it should offer a way to inject as well, but that goes a bit beyond the scope of this ticket).

Reply:
Should go with the LinkedHashModelFactory instead then. Since it's usually faster.